react/week1/my-app/src/components/todoList.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Todo from "./todo";
+
+const TodoList = ({ todoList }) => {
+  return (
+    <div>
+      {todoList.map((todo) => {
+        return <Todo todo={todo} />;
```

Comment:
Here you need to pass a key prop as well with unique ID as a value. This will allow React to keep track of changes in your list.